cases/models.py

```diff
@@ -427,26 +427,28 @@ def merged_into_final(self):
     @cached_property
     def timeline_merge_records(self):
         """Return a list of MergeRecords to display in the timeline for this case."""
-        merge_map = self.merge_map
-        mergee_records = MergeRecord.objects.filter(
-            Q(merged_into_id__in=merge_map.keys()) | Q(mergee_id__in=merge_map.keys())
+
+        # Records this case is is referenced in.
```

Comment:
Double is :)

Reply:
Also I think `merge_map` always includes the case itself (see the initial `merge_map = {c.id: c.id for c in cases}` line), so I'm not sure `direct_records` is required.